README.md

```diff
@@ -2,11 +2,13 @@
 a simple webscraping code developed in java using jsoup and ui4j libraries to download something in youtube and convert it into high quality audio format(mp3). It can be used as a song downloader.
 
 ## Requirements
-- System requirements
-
-    `apt-get install qt5-default libqt5webkit5-dev build-essential python-lxml       python-pip xvfb` 
+- System requirements  
+    - Ubuntu  
+        `apt-get install qt5-default libqt5webkit5-dev build-essential python-lxml       python-pip xvfb` 
+    - Arch Linux  
+        `pacman -S install qt5-base qt5-webkit base-devel python2-pip xorg-server-xvfb jre8-openjdk`  
```

Comment:
You have included `install` as one of the packages for installation. `pacman` would throw an error failing to find package `install`.

    error: target not found: install